docs/content_management/create_edit_pages.md

```diff
@@ -77,33 +96,72 @@ Each block is unique in its function and enables you to customize the content ap
+You can reorder blocks in a few ways:
+
+- drag and drop block in the desired location on a drop zone
+- access Structure view and use 'Move up' and 'Move down' function in the settings of the block
+
+When you add a new block to the drop zone, drop it in the blue highlighted area. Before you drop it, a bold line appears  - it helps you see the position of the newly added block in relation to other, already added blocks.
+
+![Drop zone line](drop_zone_line.png)
+
+When you add a block by dragging it from Elements menu into the drop zone,
+the block settings panel open immediately where you can configure all block properties.
+
+![Block properties](block_properties.png)
+
+This is a default behavior. You can globally turn off automatic opening of the block settings panel in the user settings.
+First, access your user settings on the right side of the top bar:
+
+![Elements menu](img/user_settings.png "User settings")
+
+Then, go to **My preferences** tab, **Edit** section.
+Here, you can find `Automatically open block settings in builder` setting, which, by default, is set up to `enabled` value.
+To change this behavior, click on **Edit** icon, find the setting, and change its value to `disabled`.
+
+![User settings - blocks](img/user_settings_blocks.png "User settings - blocks settings")
 
 [[= product_name =]] comes with a set of ready-to-use Page blocks,
 but [developers can add their own]([[= developer_doc =]]/content_management/pages/create_custom_page_block/).
 
-![Elements menu](img/elements_menu.png "Elements menu")
-
 For a list of blocks available out of the box, see [Block reference](block_reference.md).
 
-!!! note 
+!!! note
 
-    Before you add a block that involves products, product types, or product categories, make 
-    sure your that your [user Role](../permission_management/permissions_and_users.md) has 
+    Before you add a block that involves products, product types, or product categories, make
+    sure your that your [user Role](../permission_management/permissions_and_users.md) has
     the `Product/View` and `Product type/View` permission.
 
     Before you add a block that uses results derived from a [Personalization](../personalization/personalization.md) scenario, 
-    for example, [Dynamic targeting](block_reference.md#dynamic-targeting-block) or [Recently added block](block_reference.md#recently-added-block), make sure that the scenario is 
-    [properly configured](../personalization/configure_scenarios.md).
+    for example, [Dynamic targeting](block_reference.md#dynamic-targeting-block) or [Recently added block](block_reference.md#recently-added-block), make sure that the scenario is [properly configured](../personalization/configure_scenarios.md).
+
+You can work with the block, for example, duplicate it, change its position or delete it.
+To access options menu, hover over the block - the toolbar appears.
+Click the icon to see more options.
```

Comment:
```suggestion
To access the actions, hover over the block, and the toolbar appears.
Some of them are available after you click the **Options** icon.
```
I've found a name for this icon in figma. It is "options, more, three dots"

Reply:
To be more precised: it's called kebab :)

Reply:
"the three dots", as discussed with @juskora.